pkg/api/pod/util.go

```diff
@@ -390,6 +390,37 @@ func usesIndivisibleHugePagesValues(podSpec *api.PodSpec) bool {
 	return false
 }
 
+// isSameExpandedDNSConfig returns true if the oldPodSpec already had
+// ExpandedDNSConfig and podSpec has the same DNSConfig
+func isSameExpandedDNSConfig(podSpec, oldPodSpec *api.PodSpec) bool {
```

Comment:
should we add an unit test for this function to protect this code?

Reply:
Added a unit test.
I think this function can be removed after the alpha stage(?).
Until then, it would be nice to test the functionality.